django_x509/tests/test_ca.py

```diff
@@ -681,3 +681,8 @@ def test_ca_common_name_length(self):
         message_dict = context_manager.exception.message_dict
         self.assertIn('common_name', message_dict)
         self.assertEqual(message_dict['common_name'][0], msg)
+
+    def test_ca_without_key_length_and_digest_algo(self):
+
+        with self.assertRaises(ValidationError):
+            self._create_ca(key_length=None, digest=None)
```

Comment:
We need to also check that validation error is being raised for right fields. Check https://github.com/openwisp/openwisp-network-topology/pull/134/files#diff-680c7234a86452af223e255468e468e9d041aa110601b06115818e285b8cccd9R353-R357 for reference.